sigproc/src/OmnibusSigProc.cxx

```diff
@@ -1464,7 +1464,8 @@ bool OmnibusSigProc::operator()(const input_pointer& in, output_pointer& out)
             if (och.plane < 0) {
                 continue;  // in case user gives us multi apa frame
             }
-            m_cmm[name][och.channel] = m.second;
+            // m_cmm[name][och.channel] = m.second;
+            m_cmm["bad"][och.channel] = m.second; // make sure load_data gets all the masked channels
```

Comment:
the assumption here is to add all cmm to bad?
and don't change to och.channel for OmnibusSigProc output cmm?

Reply:
the "bad" label is hardcoded in several other places in the module, and so changing all cmm to "bad" for the cmm inside SigProc seemed like the simplest change. I'm happy to discuss other options though!